airflow/models/baseoperator.py

```diff
@@ -975,7 +965,6 @@ def __init__(
         if self.pool_slots < 1:
             dag_str = f" in dag {dag.dag_id}" if dag else ""
             raise ValueError(f"pool slots for {self.task_id}{dag_str} cannot be less than 1")
-        self.sla = sla
```

Comment:
shouldn't we add here also a deprecation warning like you added here?
https://github.com/apache/airflow/pull/42285/files#diff-62c8e300ee91e0d59f81e0ea5d30834f04db71ae74f2e155a10b51056b00b59bR639

Reply:
I considered it, but then I think we'll be getting a lot of duplicate warnings.  I think in one place should be good?

Reply:
because todays sla is also for tasks so we should add a deprecation warning here also.

Reply:
I feel like that will be super noisy, but I can add it.

Reply:
Added in c712b60